lib/iris/experimental/equalise_cubes.py

```diff
@@ -8,38 +8,23 @@
 
 """
 
-import numpy as np
-
 
 def equalise_attributes(cubes):
     """
     Delete cube attributes that are not identical over all cubes in a group.
 
-    This function simply deletes any attributes which are not the same for
-    all the given cubes.  The cubes will then have identical attributes.  The
-    given cubes are modified in-place.
+    .. warning::
 
-    Args:
+        This function is now **disabled**.
 
-    * cubes (iterable of :class:`iris.cube.Cube`):
-        A collection of cubes to compare and adjust.
+        The functionality has been moved to
+        :func:`iris.util.equalise_attributes`.
 
     """
-    # Work out which attributes are identical across all the cubes.
-    common_keys = list(cubes[0].attributes.keys())
-    for cube in cubes[1:]:
-        cube_keys = list(cube.attributes.keys())
-        common_keys = [
-            key
-            for key in common_keys
-            if (
-                key in cube_keys
-                and np.all(cube.attributes[key] == cubes[0].attributes[key])
-            )
-        ]
-
-    # Remove all the other attributes.
-    for cube in cubes:
-        for key in list(cube.attributes.keys()):
-            if key not in common_keys:
-                del cube.attributes[key]
+    old = "iris.experimental.equalise_cubes.equalise_attributes"
+    new = "iris.util.equalise_attributes"
+    emsg = (
+        f'The function "{old}" has been moved.\n'
+        f'Please replace "{old}(<cubes>)" with "{new}(<cubes>)".'
```

Comment:
Ohhh our first `f-string` 🎉 

Welcome to the wonderful world of Python3 (finally!)